db/models.py

```diff
@@ -139,6 +139,7 @@ class EVCBaseDoc(DocumentBaseModel):
     metadata: Dict = {}
     active: bool
     enabled: bool
+    error_status: bool = False
```

Comment:
`error_status: bool` can work, but notice that it's not an optimal choice and make it very difficult to evolve later, plus it's not also aligned with the rest of code base where we've standardized to report errors as a set|list of strs (when the error can get accumulated) like status_reason or just a string. Check out my comment below to see if it makes sense. 

Plus, if data model gets augmented let's not forget DB scripts to backfill, this wouldn't break things but we wan't to have the documents with backfilled values too to avoid confusion. But, this can be handled subsequently before the feature gets completely delivered.

Reply:
Changed to a dictionary. It an error happens it would populated with `{'current_path': 'install'}` indicating an error happened when installing `current_path`.

Reply:
Deleted `error_status`, response from error installation and deletion is same as it was before.